frontend/src/response.rs

```diff
@@ -0,0 +1,33 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize)]
+#[serde()]
+pub struct SpecificResponse {
+    pub compile: String,
+    pub run: String,
+}
+#[derive(Serialize, Deserialize)]
+#[serde()]
+pub struct Res {
+    pub errors: SpecificResponse,
```

Comment:
tę strukturę można by lepiej zaprojektować, bo jeśli jest compile error, to przecież nie będzie outputu z runtime'u